test/integration/features/cmd-openshift.feature

```diff
@@ -5,143 +5,164 @@ cluster in VM provided by Minishift.
+      And stdout should match
+      """
+      http:\/\/nodejs-ex-myproject\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.nip\.io
+      """
 
   Scenario: Getting URL of service using --url flag
-      When executing "minishift openshift service nodejs-ex --url" succeeds
-      Then stdout should be valid URL
+     When executing "minishift openshift service nodejs-ex --url" succeeds
+     Then stdout should be valid URL
 
   Scenario: Deployed Node.js application is available
-      When service "nodejs-ex" rollout successfully within "1200" seconds
-      Then with up to "30" retries with wait period of "1000ms" the "body" of HTTP request to "/" of service "nodejs-ex" in namespace "myproject" contains "Welcome to your Node.js application on OpenShift"
-       And with up to "30" retries with wait period of "1000ms" the "status code" of HTTP request to "/" of service "nodejs-ex" in namespace "myproject" is equal to "200"
+     When service "nodejs-ex" rollout successfully within "1200" seconds
+     Then with up to "30" retries with wait period of "1000ms" the "body" of HTTP request to "/" of service "nodejs-ex" in namespace "myproject" contains "Welcome to your Node.js application on OpenShift"
+      And with up to "30" retries with wait period of "1000ms" the "status code" of HTTP request to "/" of service "nodejs-ex" in namespace "myproject" is equal to "200"
 
   Scenario: Seeing configuration of OpenShift master
   Minishift openshift config view prints YAML configuration of OpenShift cluster.
   Note: --target=master is default value for minishift openshift config command
-      When executing "minishift openshift config view" succeeds
-      Then stdout should be valid YAML
+     When executing "minishift openshift config view" succeeds
+     Then stdout should be valid YAML
 
   Scenario: Seeing configuration of OpenShift node
-      When executing "minishift openshift config view --target node" succeeds
-      Then stdout should be valid YAML
+     When executing "minishift openshift config view --target node" succeeds
+     Then stdout should be valid YAML
 
   Scenario: Setting configuration on OpenShift master
-      When executing "minishift openshift config set --patch '{"assetConfig": {"logoutURL": "http://www.minishift.io"}}'" succeeds
-      Then stdout should contain "Patching OpenShift configuration"
-       And user waits "30" seconds
-      When executing "minishift openshift config view" succeeds
-      Then stdout is YAML which contains key "assetConfig.logoutURL" with value matching "http://www\.minishift\.io"
-
-  Scenario: Add a component to running openshift cluster
-      When executing "minishift openshift component add service-catalog" succeeds
-       And stdout should contain
-       """
-       Finished installing "openshift-service-catalog"
-       """
+     When executing "minishift openshift config set --patch '{"assetConfig": {"logoutURL": "http://www.minishift.io"}}'" succeeds
```

Comment:
Have tried this on Windows? I am suspicious that it will have troubles here.

Reply:
@agajdosi This is not a new scenario as @amitkrout has fixed the formatting, so wondering why we have not reported if it fails in Windows?

Reply:
@LalatenduMohanty I see, missed that. So there is no problem with this. Thanks for mentioning.

Reply:
I have used same file for downstream bits validation and never come across with the failure on windows terminal (powershell)

Reply:
@amitkrout Yes, I have missed that it is already added and thus it works as we tested the feature over and over. Thought it is new scenario...